mmv1/third_party/terraform/tests/resource_billing_budget_test.go

```diff
@@ -241,6 +241,7 @@ resource "google_billing_budget" "budget" {
     labels  = {
       label1 = "bar2"
     }
+	services = ["services/24E6-581D-38E5"] # Bigquery
```

Comment:
It looks like the indentation may be off? Probably doesn't break anything but would be nice to fix.

Reply:
Fixed! Thanks for catching it!